### PR TITLE
use OS packaged version if the hoe gem

### PR DESCRIPTION
**Pull request by planthaber**
Hoe gem fails recently because of version conflicts with rake:

on master:
in directory /data/jenkins/workspace/workspace/rock-basic/Flavor/master/node/Ubuntu_15.04/dev/base/scripts
    cannot load the Hoe gem. Distribution is disabled

on stable:
'utilrb' cannot be build -- loading gem failed: Unable to activate hoe-3.14.2, because rake-11.1.1 conflicts with rake (< 11.0, >= 0.8)

using the packages version solves the issue

### CODE REVIEW

**Comment by jmachowinski** [comment]
is this commit still needed ?

**Comment by planthaber** [comment]
I guess so, the problem is that a lot of current versions gems are incompatible to ruby 2.0, thisd also applies to e.g. rake, as long autoproj installs them this is fine as you can set the version to be installed, in case it is installed via rubygems dependency gem install takes the current version, no matter if compatible or not.

The only way around this would be to install the gem with the lower version BEFORE the dependency mechanism of gem install tries to install it (gem already available). In autoproj 1.9 there is no way to do this than using the debian packages, which should always be compatible . 

long story short: the issue was with rake , so the rake-compiler debian package should be installed to fix this.
